### ISSUE TITLE
Icingaweb2 RPM places apache user in icingaweb2 group

### ISSUE THREAD

**Issue by basg** [Icinga/icinga-packaging#188]
It seems that RPM package `icingaweb2` adds the user `apache` to groups `icingacmd` and `icingaweb2`. 

```
$ rpm -q --scripts icingaweb2
preinstall scriptlet (using /bin/sh):
getent group icingacmd >/dev/null || groupadd -r icingacmd
usermod -a -G icingacmd,icingaweb2 apache
exit 0
```

If I'm correct, this should not be necessary, and may even impose a security risk.

**Comment by htriem** [comment]
We need to test and verify this.

**Comment by prupert** [comment]
Yes, please, there is absolutely no need for this. It may be an artifact from legacy mod_php setups using cmdpipe?

**Comment by yhabteab** [comment]
> Yes, please, there is absolutely no need for this. It may be an artifact from legacy mod_php setups using cmdpipe?

It's not an artefact, all recent packages still add the Apache user to that group, as there may still be someone out there using `cmdpipe` instead of the API, even if it's already deprecated. However, we should consider in removing this from future releases, and if someone still uses `cmdpipe`, they will have to add it to that group themselves.

**Comment by Al2Klimov** [comment]
mod_php is absolutely not legacy as long as [we use it](https://github.com/Icinga/docker-icingaweb2/blob/master/Dockerfile) in Docker.

Regardless IW2 runs on mod_php or FPM, the user executing PHP code [must be](https://github.com/Icinga/icingaweb2/blob/v2.12.6/modules/setup/application/views/scripts/form/setup-welcome.phtml#L35) in the icingaweb2 group.

icingacmd just gives access to the command pipe if any. So what... 🤷‍♂️
